circle.yml

```diff
@@ -689,6 +689,18 @@ commands:
             scripts/notify-slack.sh
           fi
 
+  notify-slack-master-failure:
+    steps:
+    - run:
+        name: Send a Slack notification on a master branch failure
```

Comment:
GitHub has an orb for this now (but also OK with the script):
https://circleci.com/orbs/registry/orb/circleci/slack

Reply:
Nice, this looks like it would be worth switching to in the future.